src/screens/Sidebar/ReportBugs.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { View, WebView, ActivityIndicator } from 'react-native';
+import { connect } from 'react-redux';
+import { cannyBoardTokens } from 'kitsu/constants/app';
+import PropTypes from 'prop-types';
+import styles from './styles';
+
+class ReportBugs extends React.Component {
+  static navigationOptions = {
+    title: 'Report Bugs',
+  };
+
+  state = {
+    loading: true,
+  };
+
+  componentDidMount() {
+    this.getCannySsoKey();
+  }
+
+  getCannySsoKey = async () => {
+    const { accessToken } = this.props;
+    fetch('https://kitsu.io/api/edge/sso/canny', {
+      method: 'GET',
+      headers: {
+        'Accept': 'application/vnd.api+json',
+        'Content-Type': 'application/vnd.api+json',
+        'Authorization': `Bearer ${accessToken}`,
+        'Host': 'kitsu.io',
```

Comment:
I think we can remove the `Content-Type` and `Host` properties here, and then use named keys rather than strings.